src/utils.py

```diff
@@ -298,6 +308,7 @@ def coco_evaluation(gt_filepath, prediction_filepath, image_ids, category_ids, s
     cocoEval.params.areaRng = [[0 ** 2, 1e5 ** 2], [0 ** 2, small_annotations_size ** 2],
                                [small_annotations_size ** 2, 1e5 ** 2]]
     cocoEval.params.areaRngLbl = ['all', 'small', 'large']
+    cocoEval.params.maxDets = [1, 10, 100000]
```

Comment:
@taraspiotr Thanks. I put it here because I found it somewhere else in code. I'd better remove this line.

Reply:
@apyskir True, I believe I was using it for testing recall with multilayer without NMS, here it shouldn't matter, because after NMS there shouldn't be over 100 annotations for one image.